README.md

```diff
@@ -178,3 +178,56 @@ bucket into a publicly browsable file tree. Useful if you want something
 like Apache's directory index for your software download repository.
 Plume release handles this as well, so it does not need to be run as part of
 the release process.
+
+## Cloud Provider Credentials
+Each platform reads the credentials it uses from different files.
+
+### qemu
+`qemu` is run locally and needs no credentials, but does need to be run as root.
+
+### aws
+`aws` uses the `~/.aws/credentials` file to store credentials. This can be populated manually:
```

Comment:
Should probably state in the first sentence of a section when config files share a format with other tools.